src/main/java/net/minecraftforge/items/pickup/ItemPickupQuery.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.items.pickup;
+
+import org.jetbrains.annotations.NotNull;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.entity.item.ItemEntity;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.Level;
+
+import java.util.Objects;
+import java.util.function.Predicate;
+
+/**
+ * FunctionalInterface used to query the Level to determine if the specified ItemEntity is valid to be picked up or not.
+ */
+@FunctionalInterface
+public interface ItemPickupQuery
```

Comment:
The javadoc should clarify what "valid to be picked up or not" actually means. What does returning `true` or `false` in the interface's method mean for the pickup logic?